downstream/modules/platform/ref-general-inventory-variables.adoc

```diff
@@ -0,0 +1,27 @@
+[id="ref-general-inventory-variables"]
+
+= General variables
+
+[cols="50%,50%",options="header"]
+|====
+| *Variable* | *Description*
+| *`registry_password`* | Password credential for access to `registry_url`.
+
+Used for both `[automation controller]` and `[automation hub]` groups.
```

Comment:
`[automationcontroller]` (without space) - also in registry_url and registry_usernmae.